torchdata/dataloader2/communication/eventloop.py

```diff
@@ -32,36 +31,42 @@
     "DataPipeToQueuesLoop",
     "CreateProcessForDataPipeline",
     "CreateProcessForMultipleDataPipelines",
-    "CreateThreadForDataPipeline",
 ]
 
 
-class _ResetCounter:
+class _RequestCounter:
```

Comment:
nit: I think it would be nice to have a short documentation about the usage of this class.

Reply:
Sure